BitwardenShared/UI/Platform/Settings/SettingsCoordinator.swift

```diff
@@ -173,9 +173,11 @@ final class SettingsCoordinator: Coordinator, HasStackNavigator { // swiftlint:d
     ///
     private func showAbout() {
         let processor = AboutProcessor(
+            aboutAdditionalInfo: DefaultAboutAdditionalInfo(),
             coordinator: asAnyCoordinator(),
             services: services,
-            state: AboutState()
+            state: AboutState(),
+            systemDevice: UIDevice.current
```

Comment:
🤔 There is a `HasSystemDevice` already on the `Services`; would it be possible to pass that in here?

Reply:
oh missed that, it's a really good idea thanks!